projects/memapCore/src/main/java/memap/messages/planning/StorageMessage.java

```diff
@@ -22,6 +22,12 @@
 	public double efficiencyCharge;
 	public double efficiencyDischarge;
 	
+	public double storageLosses;
+	
+	public double alpha;
```

Comment:
See also comment in the CVSStorage class:



The storageMessage is evisioned to contain only "device-parameters". 

Alpha and betas are calculated in the MPC part, hence, they should not be part of the storage message.

We sould even stick to the standard DIN EN 12897, which has the "Warmhalteverlust in W" label.

Reply:
I agree with sticking to Watt for the standby losses as input value by user, as this is also according to DIN EN 12897. We should just have a look, if this is also the appropriate unit for batteries.

I would not do an enum or a case distinction, as for other input parameters we also force the user to insert the values in the correct units specified by us.

However, independent from the unit to be entered, we would need an additional reference SOC at which this standby losses are measured. For DIN EN 12897 it is SOC_ref = 0.75 (assuming that T_max is 80°C and T_min is 20°C). But should we just assume that the entered value (in Watt) always refers to the reference SOC of 0.75 or make an additional input? Perhaps for the sake of simplicity and due to time issues we should assume 0.75 as hardcoded standard reference SOC?